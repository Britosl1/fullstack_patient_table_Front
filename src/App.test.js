import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

describe('Snapshot test', () => {
  it('renders correctly', () => {
    const app = renderer
      .create(<App />)
      .toJSON();
    expect(app).toMatchSnapshot();
  });

})
