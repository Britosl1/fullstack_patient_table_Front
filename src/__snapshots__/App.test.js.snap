// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test renders correctly 1`] = `
<div
  className="sc-crHmcD iBZNmr"
  color="#264653"
>
  <nav
    className="sc-bdvvtL dcNLmR"
    color="#E76F51"
  >
    <h1
      className="sc-gsDKAQ cRtoSf"
      color="#F4A261"
    >
      Fullstack Form Project
    </h1>
    <img
      className="sc-dkPtRN bVjsqg"
      src="profile.png"
    />
  </nav>
  <div
    style={
      Object {
        "padding": "50px",
      }
    }
  >
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="Search for user name or nationality..."
      style={
        Object {
          "marginBottom": "20px",
        }
      }
      type="text"
      value=""
    />
    <div
      className="table-responsive"
    >
      <table
        className="table table-dark table-sm table-striped table-bordered table-hover"
      >
        <thead>
          <tr>
            <th>
              Id
            </th>
            <th>
              Full Name
            </th>
            <th>
              Email
            </th>
            <th>
              location
            </th>
            <th>
              Birth
            </th>
            <th>
              Gender
            </th>
            <th>
              Phone
            </th>
            <th>
              Nationality
            </th>
          </tr>
        </thead>
        <tbody />
      </table>
    </div>
    <button
      className="sc-hKwDye bzhAWL"
      color="#E76F51"
      onClick={[Function]}
    >
      <p
        className="sc-eCImPb bjUvSA"
        color="#264653"
      >
        Show More
      </p>
    </button>
    <div>
      <ul
        className="pagination"
      >
        <li
          className="page-item active"
        >
          <span
            className="page-link"
            disabled={false}
          >
            1
            <span
              className="visually-hidden"
            >
              (current)
            </span>
          </span>
        </li>
        <li
          className="page-item"
        >
          <a
            className="page-link"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            2
          </a>
        </li>
        <li
          className="page-item"
        >
          <a
            className="page-link"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            3
          </a>
        </li>
        <li
          className="page-item"
        >
          <a
            className="page-link"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            4
          </a>
        </li>
        <li
          className="page-item"
        >
          <a
            className="page-link"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="button"
            tabIndex={0}
          >
            5
          </a>
        </li>
      </ul>
      <br />
    </div>
  </div>
</div>
`;
